### PR TITLE
Update README.md task 4

### PR DESCRIPTION
**Pull request by jalas167**

### CODE REVIEW

File: tasks/4-text-2-speech-server/README.md [hunk 1]
```diff
@@ -48,10 +48,10 @@ This allows the REST API to respond more quickly to new requests.
 
 ### Useful links to solve this task
 
-- https://docs.golem.network/creators/javascript/high-level/task-model/
-- https://docs.golem.network/creators/javascript/examples/hello/
-- https://docs.golem.network/creators/javascript/examples/simple/
-- https://docs.golem.network/creators/javascript/examples/hashcat/
+- https://docs.golem.network/docs/creators/javascript/high-level/task-model/
```
Comment (cryptobench) [inline tasks/4-text-2-speech-server/README.md:51]: Broken link

File: tasks/4-text-2-speech-server/README.md [hunk 2]
```diff
@@ -48,10 +48,10 @@ This allows the REST API to respond more quickly to new requests.
...
+- https://docs.golem.network/docs/creators/javascript/examples/hello/
```
Comment (cryptobench) [inline tasks/4-text-2-speech-server/README.md:52]: Broken link

File: tasks/4-text-2-speech-server/README.md [hunk 3]
```diff
@@ -48,10 +48,10 @@ This allows the REST API to respond more quickly to new requests.
...
+- https://docs.golem.network/docs/creators/javascript/examples/simple/
+- https://docs.golem.network/docs/creators/javascript/examples/hashcat/
```
Comment (cryptobench) [inline tasks/4-text-2-speech-server/README.md:54]: Broken link

File: tasks/4-text-2-speech-server/README.md [hunk 4]
```diff
@@ -48,10 +48,10 @@ This allows the REST API to respond more quickly to new requests.
 
 ### Useful links to solve this task
 
-- https://docs.golem.network/creators/javascript/high-level/task-model/
-- https://docs.golem.network/creators/javascript/examples/hello/
-- https://docs.golem.network/creators/javascript/examples/simple/
-- https://docs.golem.network/creators/javascript/examples/hashcat/
+- https://docs.golem.network/docs/creators/javascript/high-level/task-model/
+- https://docs.golem.network/docs/creators/javascript/examples/hello/
+- https://docs.golem.network/docs/creators/javascript/examples/simple/
```
Comment (cryptobench) [inline tasks/4-text-2-speech-server/README.md:53]: Broken link

File: tasks/4-text-2-speech-server/README.md [hunk 5]
```diff
@@ -93,8 +93,8 @@ can be found at **https://preview.golem.network/**
 
 ## Useful links:
 
-- Javascript docs: https://docs.golem.network/creators/javascript/quickstart/
-- Dapps docs: https://docs.golem.network/creators/dapps/ 
+- Javascript docs: https://docs.golem.network/docs/creators/javascript/quickstart/
```
Comment (cryptobench) [inline tasks/4-text-2-speech-server/README.md:96]: Broken link, should be https://docs.golem.network/docs/creators/javascript/quickstarts/quickstart

**Review by cryptobench — CHANGES_REQUESTED** [review]

**Comment by jalas167** [comment]
will not patch, as mid-level api docs are no longer available, and removing the links would remove the context

**Pull request closed without merging**